.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+        branches: ['**']
+
+jobs:
+    build-checks:
+        name: Set-up, Lint, Format and Build
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Set up Node.js
+              uses: actions/checkout@v4
+              with:
+                  node-version-file: '.nvmrc'
```

actionlint (1.7.7)

input "node-version-file" is not defined in action "actions/checkout@v4". available inputs are "clean", "fetch-depth", "fetch-tags", "filter", "github-server-url", "lfs", "path", "persist-credentials", "ref", "repository", "set-safe-directory", "show-progress", "sparse-checkout", "sparse-checkout-cone-mode", "ssh-key", "ssh-known-hosts", "ssh-strict", "ssh-user", "submodules", "token"

(action)